app/src/main/res/layout/edit_text_dialog.xml

```diff
@@ -10,6 +10,6 @@
             android:layout_height="wrap_content"
             android:inputType="text"
             android:ems="10"
-            android:id="@+id/text" />
+            android:id="@+id/urlEditText" />
```

Comment:
That layout is also used in the edit feed dialog, which now crashes because it does not find the view. Could you please migrate that dialog to ViewBinding, too?

Reply:
I'm sorry, I actually didn't run this on emulator. Will check and update soon

Reply:
Sorry, I have to be more precise. It's in the _rename_ feed dialog